src/cardanoOutputs.h

```diff
@@ -0,0 +1,18 @@
+#ifndef H_CARDANO_APP_CARDANO_OUTPUTS
+#define H_CARDANO_APP_CARDANO_OUTPUTS
+
+#include "cardano.h"
+
+
+typedef struct {
+	uint8_t policyId[MINTING_POLICY_ID_SIZE];
+} token_group_t;
+
+typedef struct {
+	uint8_t assetName[ASSET_NAME_SIZE_MAX];
```

Comment:
assetNameBytes is imho a better name to not mislead anybody into thinking it's a (null-terminated) string